wren-core-py/Cargo.toml

```diff
@@ -9,19 +9,23 @@ name = "wren_core_py"
 crate-type = ["cdylib"]
 
 [dependencies]
-pyo3 = { version = "0.21.2", features = [
-  "extension-module",
-  "generate-import-lib",
-] }
+pyo3 = { version = "0.23.2", features = ["extension-module"] }
 wren-core = { path = "../wren-core/core" }
 base64 = "0.22.1"
 serde_json = "1.0.117"
-thiserror = "1.0"
+thiserror = "2.0.3"
 csv = "1.3.0"
 serde = { version = "1.0.210", features = ["derive"] }
 env_logger = "0.11.5"
 log = "0.4.22"
 tokio = "1.40.0"
 
+[dev-dependencies]
+rstest = "0.23.0"
+
 [build-dependencies]
-pyo3-build-config = "0.21.2"
+pyo3-build-config = "0.23.2"
+
+[features]
+extension-module = ["pyo3/extension-module"]
+default = ["extension-module"]
```

Comment:
I'm not sure but why do we need this feature flag? 🤔  I tried to remove them and everything is fine.

Reply:
Follow this [FAQ](https://pyo3.rs/v0.22.5/faq#i-cant-run-cargo-test-or-i-cant-build-in-a-cargo-workspace-im-having-linker-issues-like-symbol-not-found-or-undefined-reference-to-_pyexc_systemerror)
If we add these configurations, we can test Rust without compiling to Python.